<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>57</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_kpat">
       <property name="text">
        <string>KPat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_kmines">
       <property name="text">
        <string>KMines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_programm"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_find_programm">
       <property name="text">
        <string>Найти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="camera_layout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
